.github/workflows/CI-latest.yml

```diff
@@ -0,0 +1,46 @@
+name: CI-latest
+on:
+  push:
+    branches:
+      - main
+    tags: '*'
+  pull_request:
+concurrency:
+  # Skip intermediate builds: always.
+  # Cancel intermediate builds: only if it is a pull request build.
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        # we test on all versions back to LTS because we want to test for stability across
+        # julia versions
+        version:
+          - 'nightly'
```

Comment:
My goal here was to be able to have a badge in the README that shows whether or not the stable versions of Julia pass, and this was the easiest way I found of doing that. If there is some way to accomplish that and keep the CI for latest in the same yaml, happy to do that!

Reply:
and I guess you don't want the badge say failing if the stable jobs are passing?

I think this isn't necessary if https://github.com/JuliaRegistries/General/pull/98962 is merged though right?

Reply:
I think that for this specific problem, the PR to General registry solves the issue.

I guess the question is whether, in general, once #46 merges, StableHashTraits should always upper bound the version of Julia it requires, or if it is appropriate to check on nightly CI and always make sure the bugs found there get fixed before that becomes a stable release.

It would be nice to run on latest but seems safer to upper bound Julia's version: I guess I could set the upper bound and then in the "latest" CI explicitly change the Project.toml to remove the upper bound and run the test.

Reply:
For now I'm going to punt on this. I'll remove the CI here, and mark this as an issue (#47).